FirstProject/assets/feedback.less

```diff
@@ -0,0 +1,252 @@
+/*Feedback HTML*/
+
+//Import colors file
+@import 'colors';
+
+/*Body section*/
+body {
+  overflow-x:hidden;
+}
+
+.left-image {
+  height: 290px;
+}
+
+.right-image {
+  float: right;
+}
+
+.right-title {
+  font-weight: bold;
+  margin-top:50px;
+}
+
+.container-fluid {
+  background-color: lightgrey;
+}
+
+.feedback-image {
+  width: 70%;
+}
+
+a {
+  text-decoration: none;
+  cursor: pointer;
+  color: black;
+  font-weight: 700;
+  font-size: 24px;
+}
+
+.footer-section {
+  padding: 25px 0 19px 0;
+  font-weight:bold;
+  font-size:16px;
+  color: @grey;
+  text-align: right;
+}
+
+.feedback-interview-section {
+  text-align: center;
+  padding-top: 125px;
+  position: absolute;
+  width:100%;
+}
+
+.header-form-section {
+  width: 18%;
+  background-color: @white;
+  height: 340px;
+  margin-left: 429px;
+}
+
+.header-background-section {
+  height: 640px;
+  background-image: url(../img/feedback-banner.jpg);
+  width: 100%;
+}
+
+.interview-text-area {
+  background-color: @lightBlue;
+  color: @white;
+  font-size: 22px;
+  padding: 27px;
+}
+
+.username-area-input, .password-area-input {
+  width: 80%;
+  margin-top: 30px;
+  border-right: none;
+  border-left: none;
+  border-top: none;
+}
+
+.login-button {
+  margin-top:80px;
+  padding: 3px 30px 3px 30px;
+  border: 1px solid @lightBlue;
+  color: @white;
+  background-color: @lightBlue;
+}
+
+.text-description {
+  font-size: 13px;
+  margin-top: 60px;
+}
+
+.text-description-2 {
+  margin-top:150px;
+}
+
+.footer-margin {
+  margin-top: 120px;
+}
+
+.footer-section{
+  text-align: center;
+  background-color: @lightGrey;
+  margin: 0;
+}
+
+.footer-padding {
+  padding-right: 0;
+  padding-left: 0;
+}
+
+/*Adding style for mobile devices/ tablets / large monitors*/
+
+/*Screen between 320px and 520px*/
+@media screen and (max-width: 520px) and (min-width: 320px) {
+  /* Styles */
+
+  .footer-section{
+    text-align: center;
+  }
+
+  .feedback-image {
+    width: 100%;
+  }
+
+  header form {
+    margin-left: 0!important;
```

Comment:
Why is this so `!important`? This is particularly troublesome, since it applies to _all_ `<form>` elements that happen to be part of a `<header>` element. Granted, this might not be common occurrence, but we still want to make sure that we keep our styling for generic tags... well... generic. :smile: